### PR TITLE
refactor activeDeadlineSeconds to target-level

### PR DESCRIPTION
**Pull request by ColdsteelRail**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by ColdsteelRail** [comment]
refactor activeDeadlineSeconds from job-level to target-level configuration

the TTL seconds is still job-level configuration

File: charts/templates/crd/apps.kusionstack.io_operationjobs.yaml [hunk 1]
```diff
@@ -92,7 +92,7 @@ spec:
           status:
             description: OperationJobStatus defines the observed state of OperationJob
             properties:
-              endTimestamp:
```
Comment (shaofan-hs) [inline charts/templates/crd/apps.kusionstack.io_operationjobs.yaml:95]: Is there a user using the old field, and is it necessary to ensure compatibility for this field?

File: pkg/controllers/operationjob/operationjob_controller.go [hunk 2]
```diff
@@ -160,19 +155,24 @@ func (r *ReconcileOperationJob) doReconcile(ctx context.Context, instance *appsv
 	getErr := r.getTargetsOpsStatus(ctx, actionHandler, selectedCandidates, enablePodOpsLifecycle, instance)
 	// calculate opsStatus of all candidates
 	instance.Status = r.calculateStatus(instance, candidates)
-	return controllerutils.AggregateErrors([]error{opsErr, getErr})
+	// update operationjob status
+	updateErr := r.updateStatus(ctx, instance)
```
Comment (shaofan-hs) [inline pkg/controllers/operationjob/operationjob_controller.go:159]: If getErr is not nil, we should not update the status of opj?

File: charts/templates/crd/apps.kusionstack.io_operationjobs.yaml (hunk 1 above)
Comment (ColdsteelRail) [reply charts/templates/crd/apps.kusionstack.io_operationjobs.yaml:95]: currently there is no opjs in clusters

File: pkg/controllers/operationjob/operationjob_controller.go (hunk 2 above)
Comment (ColdsteelRail) [reply pkg/controllers/operationjob/operationjob_controller.go:159]: we should always update status to show details in opj

**Review by wu8685 — APPROVED** [review]
lgtm

**Pull request merged**